models/Farm.ts

```diff
@@ -0,0 +1,14 @@
+import { Schema, model } from "mongoose";
+
+import { IFarm } from "../interfaces/farm.interface";
+
+const Farm = model<IFarm>(
+  "Farm",
+  new Schema({
+    name: { type: String, required: true },
+    address: { type: String, required: true },
```

Comment:
let's make the address and userIds not required, so you can create the farm with the name in the signUp

Reply:
I think the address can be optional.... but userId can be required as based on a user we are creating the farm...

Reply:
both optionals, so when you create the Farm inside the signup it won't be a problem